.circleci/config.yml

```diff
@@ -28,6 +28,35 @@ apt-run:  &apt-install
       libgeos-dev \
       otf-freefont
 
+conda36-run:  &conda36-install
```

Comment:
Why switch to conda?  It looks like all we are using is python and then pip installing everything else.

Reply:
I switch to conda only for the environment: we now need to access the scripts "sphinx-build", and not only the library as before. When installing with --user, we either need to change the PYTHONPATH manually, or use conda.

Reply:
Why not just use `python -msphinx` instead?  AFAIK sphinx-build is a strict synonym to it, except that `python -msphinx` looks up PYTHONPATH (which is the same as what we did before in make.py) whereas sphinx-build uses PATH (which may be set differently).

This relates to my earlier point about using the shorter/newer makefile generated by recent versions of sphinx.